### PR TITLE
Fix DecentralandUtils.getAllAssetsForSale

### LINKED ISSUE #299: Tesnet: Fix land parcel purchase
For now, the `tasit` app is only showing estate assets because some parcels buying transactions have been reverted.
The same behavior can be reproduced using `tasit-sdk` Decentraland `should buy a parcel of land` test case (against testnet only, it's working on local blockchain).

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#299 
### Auto-close the issue?

Closes #299 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

### Notes

- `DecentrandUtils.getAllAssetsForSale` was returning overwrote and canceled sell orders.
- With this fix we can back to show parcels on the app.

### CODE REVIEW

File: packages/tasit-sdk/src/scripts/checkDecentralandContractsData.js [hunk 1]
```diff
@@ -50,42 +50,67 @@ describe("Decentraland App pre-conditions", () => {
...
-          `https://api.decentraland.org/v1/estates/${assetId}/map.png`
-        );
-        const imageData = (await image.buffer()).toString("base64");
-        expect(
-          imageData,
-          `The image of the estate (id: ${assetId}) is blank`
-        ).not.equals(blankImageData);
-      } else if (isParcel) {
-        // Note: Parcel assets always show correct image
+    it("shouldn't have no duplicated sell orders", async () => {
```
Comment (pcowgill) [inline packages/tasit-sdk/src/scripts/checkDecentralandContractsData.js]:
```suggestion
    it("shouldn't have any duplicated sell orders", async () => {
```

**Comment by pcowgill** [comment]
@marcelomorgado Nicely done! I'll merge this once the tests finish running

**Comment by pcowgill** [comment]
@marcelomorgado This test failed:

```
1) TasitAction.Contract
tasit-action:        Transactions (Actions) Subscription
tasit-action:          should emit error event when block reorganization occurs - tx confirmed twice:
tasit-action:       AssertionError: expected false to be true
tasit-action:       + expected - actual
tasit-action:       -false
tasit-action:       +true
tasit-action:       
tasit-action:       at Context.it (src/contract/Contract.test.js:500:7)
```

**Comment by marcelomorgado** [comment]
> @marcelomorgado This test failed:
> 
> ```
> 1) TasitAction.Contract
> tasit-action:        Transactions (Actions) Subscription
> tasit-action:          should emit error event when block reorganization occurs - tx confirmed twice:
> tasit-action:       AssertionError: expected false to be true
> tasit-action:       + expected - actual
> tasit-action:       -false
> tasit-action:       +true
> tasit-action:       
> tasit-action:       at Context.it (src/contract/Contract.test.js:500:7)
> ```

Looking into that now

**Comment by marcelomorgado** [comment]
I've rerun the test and it passes. Seems another non-deterministic test case.

**Comment by pcowgill** [comment]
> I've rerun the test and it passes. Seems another non-deterministic test case.

Ah ok, darn. Maybe flag that with an issue if there isn't one already?

**Comment by marcelomorgado** [comment]
> > I've rerun the test and it passes. Seems another non-deterministic test case.
> 
> Ah ok, darn. Maybe flag that with an issue if there isn't one already?

https://github.com/tasitlabs/TasitSDK/issues/329

**Pull request merged**